source/ShipInfoPanel.cpp

```diff
@@ -126,7 +128,7 @@ void ShipInfoPanel::Draw()
 bool ShipInfoPanel::KeyDown(SDL_Keycode key, Uint16 mod, const Command &command, bool isNewPress)
 {
 	bool shift = (mod & KMOD_SHIFT);
-	if(key == 'd' || key == SDLK_ESCAPE || (key == 'w' && (mod & (KMOD_CTRL | KMOD_GUI))))
+	if((key == 'd' && !shift) || key == SDLK_ESCAPE || (key == 'w' && (mod & (KMOD_CTRL | KMOD_GUI))))
```

Comment:
It is intentional that the UI button must be used to disown, and not just shift-d

Reply:
Okay, I will restore the original behavior here. (And disable the new framework tests, since mouse/pointer support is not working yet for the framework.)